scripts/cleanup.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+# Remove icons created in .tmp
+rm -rf .tmp
```

Comment:
Is this removing .tmp files all over the repository? If so, can we limit this to the `target` directory?

Reply:
It's removing the .tmp directory in root

Reply:
Right, can you remind me where in the build this is created? It's not a big deal, would be kind of nice that the build remains completely contained in `target` ...

Reply:
so the reason that .tmp is in the root as opposed to `target` is that the directory is needed at dev time as well.
It is created via a postcss plugin.